src/recommendations/RecommendationCard.jsx

```diff
@@ -20,9 +22,24 @@ const RecommendationCard = (props) => {
     return keys.slice(0, -2);
   };
 
+  const handleCardClick = () => {
+    trackRecommendationsClicked(
+      recommendation.activeRunKey,
```

Comment:
Can we convert course run key to course key?

Reply:
we are not getting course keys from Agolia.

Reply:
Yes, I know. That is why I asked to convert it. We already do this here: https://github.com/openedx/edx-platform/blob/d694b70439b9dcbb724d5954a5f7a88d05eaadd3/lms/djangoapps/learner_recommendations/views.py#L112-L113
Maybe see if prospectus already has a way of doing it otherwise you can use regex.

Reply:
yes we can do that